yolapy/models/site.py

```diff
@@ -14,4 +14,12 @@ def __init__(self, **kwargs):
     def get(cls, site_id):
         """Get a site from the Yola API."""
         site_attributes = Yola().get_site(site_id)
-        return Site(**site_attributes)
+        return cls(**site_attributes)
+
+    @classmethod
+    def for_user(cls, user_id):
```

Comment:
If you have a site id, it's expected a site exists. If you have a user id, there's no way to know if a site exists yet. For those reasons I thought they should be different methods and the user version should not raise. But I agree the inconsistent api here is weird. What if we kept this behavior but moved it to the User model? Something like `user.get_site()`?